syntax = "proto3";

package fraud;

service FraudDetection {
    rpc CheckFraud (FraudCheckRequest) returns (FraudCheckResponse);
}

message FraudCheckRequest {
    string order_json = 1;
}

message FraudCheckResponse {
    bool is_fraud = 1;
    string reason = 2;
}
